Makefile

```diff
@@ -24,14 +24,18 @@ else
 	endif
 endif
 
+.PHONY: default
```

Comment:
When the target name is not actually an output file, we should use the `.PHONY` tag. Otherwise make will try to check whether there is such a file existing, it will skip the target if there is such file existing. So, without this `.PHONY` tag, `make docs` will not work.

Adding `.PHONY` tag is a common practice in Makefiles for Go. Eg: https://github.com/helm/helm/blob/main/Makefile#L71